### PR TITLE
[Maintenance] Sync Master back to Dev when code is pushed to master

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Title - it's pointing at the `master` branch on purpose

### CODE REVIEW

**Review by dschaper — DISMISSED** [review]

**Review by DL6ER — DISMISSED** [review]

File: .github/workflows/sync-back-to-dev.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - master
+
+jobs:
+  sync-branches:
+    runs-on: ubuntu-latest
+    name: Syncing branches
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Node
```
Comment (XhmikosR) [inline .github/workflows/sync-back-to-dev.yml]: Unsure if setup-node is even needed, it shouldn't be needed AFAFICT.

Comment (PromoFaux) [reply .github/workflows/sync-back-to-dev.yml]: Did you try it? Its in the example on the actions repo.

Comment (XhmikosR) [reply .github/workflows/sync-back-to-dev.yml]: I didn't try it but it shouldn't need Node.js to be set up, AFAICT.

Comment (PromoFaux) [reply .github/workflows/sync-back-to-dev.yml]: Your hunch was correct:
https://github.com/pi-hole/land_of_confusion/runs/4772703213?check_suite_focus=true

**Comment by PromoFaux** [comment]
hahahahha. Well, it worked. But it also detected an existing PR from `master` to `development`, from a fork's `master`:

https://github.com/pi-hole/AdminLTE/pull/1813

**Pull request merged**